src/style.css

```diff
@@ -0,0 +1,143 @@
+:root {
+  --star: url(images/star.svg);
+  --star-active: url(images/star-active.svg);
+  --width: 200px;
+}
+
+* {
+  margin: 0;
```

Comment:
Don't set margin: 0 for all the selectors. Set it only for those, where you need it